nx.json

```diff
@@ -7,7 +7,15 @@
       }
     }
   },
-  "defaultBase": "master",
+  "defaultBase": "main",
```

Comment:
Was this actually necessary in the end?

Reply:
Wow, I have no recollection of this change. I don't know.